shuttle/templates/loaded_views/user_modal.html

```diff
@@ -1,3 +1,33 @@
+<!-- Add Modal -->
+    <div class="modal fade" id="edit-modal" tabindex="-1" role="dialog" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">Add {{ username }}</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    What role would you like to make {{ username }}?
+                </div>
+                <div class="dropdown">
+                    <button id="role-dropdown" class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Role
+                    </button>
+                    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                        <a class="dropdown-item" onclick="changeRole('Administrator')">Administrator</a>
+                        <a class="dropdown-item" onclick="changeRole('Driver')">Driver</a>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-primary" onclick="addUser('{{ username }}')" data-dismiss="modal">Add</button>
```

Comment:
Change this to something more descriptive for editing a user like "Save" instead of "Add"